lte/gateway/c/core/oai/test/spgw_task/spgw_test_util.h

```diff
@@ -97,5 +97,9 @@ void fill_nw_initiated_deactivate_bearer_response(
     gtpv2c_cause_value_t cause, ebi_t ebi[], unsigned int num_bearer_context,
     teid_t sgw_s11_context_teid);
 
+void fill_s11_suspend_notification(
+    itti_s11_suspend_notification_t* suspend_notif, teid_t sgw_s11_context_teid,
+    std::string imsi_str, ebi_t link_bearer_id);
```

Comment:
```suggestion
    const std::string& imsi_str, ebi_t link_bearer_id);
```

Reply:
Fixed in https://github.com/magma/magma/pull/10674/commits/bd56937b288b789ea026b02270ea5cbff5189279